src/main/java/exceptionshandling/StudentNotFoundException.java

```diff
@@ -1,4 +1,4 @@
-package ExceptionsHandling;
+package exceptionshandling;
```

Comment:
this also should be under springbootquickstarter package

Reply:
Done.